index.html

```diff
@@ -0,0 +1,241 @@
+      <div class="program-card">
+        <div class="program-card-icon">
+          <i class="fas fa-car"></i>
+        </div>
+        <h3 class="program-card-title">Boxster</h3>
+        <p class="program-card-para">The 718 Boxster takes its name from a combination.</p>
+      </div>
+
+      <div class="program-card">
+        <div class="program-card-icon">
+          <i class="fas fa-car"></i>
+        </div>
+        <h3 class="program-card-title">Cayenne</h3>
+        <p class="program-card-para">While the answer appear to be that the famous Porsche SUV.</p>
+      </div>
+
+      <div class="program-card">
+        <div class="program-card-icon">
+          <i class="fas fa-car"></i>
+        </div>
+        <h3 class="program-card-title">Carrera</h3>
+        <p class="program-card-para">A model of the 911 in modern times, the Carrera.</p>
+      </div>
+
+      <div class="program-card">
+        <div class="program-card-icon">
+          <i class="fas fa-car"></i>
+        </div>
+        <h3 class="program-card-title">Macan</h3>
+        <p class="program-card-para">The small SUV known as the “Macan”.</p>
+      </div>
+
+      <div class="program-card">
+        <div class="program-card-icon">
+          <i class="fas fa-car"></i>
+        </div>
+        <h3 class="program-card-title">Panamera</h3>
+        <p class="program-card-para">The Porsche Panamera is, as far as anyone knows.</p>
+      </div>
+
+    </div> 
+    
+    <a href="#" class="cta-text">SEE THE ENTIRE CAR GARAGE</a>
+    <button type="button" class="ctb-btn">Join Our Grand Re-opening</button>
+
+  </section>
+
+  <!-- MAIN PROGRAM SECTION ENDS -->
+
+  <!-- FEATURED SPEAKERS SECTION STARTS -->
+
+  <section class="famous-people">
+    <div class="famous-people-top">
+      <h1 class="famous-people-title">Celebrities with Porsche</h1>
+      <div class="famous-people-underline"></div>
+    </div>
+
+    <div class="famous-people-loop-holder">
+
+    </div>
+    
+  </section>
+
+  <!-- FEATURED SPEAKERS SECTION ENDS -->
+
+  <!-- PARTNERS SECTION STARTS -->
+
+  <section class="partners" id="partners-desktop">
+
+    <h2 class="partners-heading">Our Partners</h2>
+    <div class="partners-underline"></div>
+
+    <div class="partners-cards">
+
+      <div class="partners-card">
+        <img src="images/partners/totalenergies.png" class="total-energy" />
+      </div>
+
+      <div class="partners-card">
+        <img src="images/partners/netflix.png" class="netflix" />
+      </div>
+
+      <div class="partners-card">
+        <img src="images/partners/coke.png" class="Coca Cola" />
+      </div>
+
+      <div class="partners-card">
+        <img src="images/partners/facebook-1.png" class="Facebook" />
+      </div>
+
+      <div class="partners-card">
+        <img src="images/partners/google-2.png" class="Google" />
+      </div>
+
+    </div>
+
+  </section>
+
+  <!-- PARTNERS SECTION ENDS -->
+
+  <!-- FOOTER SECTION STARTS -->
+    <footer class="footer">
```

Comment:
- [ ] Kindly follow the design guidelines. In the footer, there are no navigation links, it's just the text on the right and the logo on the left 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/203852787-83e54d36-5aef-4a1b-a570-2d36d166c719.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/203852823-b4094f25-3c28-4f1e-952d-4cea6167011c.png)